### PR TITLE
docs: add usage documentation

### PR DESCRIPTION
**Pull request by douglasduteil**

### CODE REVIEW

**Comment by douglasduteil** [comment]
<p align=center> 
[Rebased]</p>

File: README.md [hunk 1]
```diff
@@ -20,6 +40,32 @@ Note, environment files will be created on *postinstall* (see [scripts/setup-env
...
+
+```sh
+# Build all the packages
+$ yarn build
+
+# Build the "code-du-travail-frontend"
+$ yarn workspace code-du-travail-frontend build
+
+# Build the "<package-name>"
+$ yarn workspace <package-name> build
```
Comment (lionelB) [inline README.md:50]: Why not put  these one (genéric) before the example

File: README.md [hunk 2]
```diff
@@ -20,6 +40,32 @@ Note, environment files will be created on *postinstall* (see [scripts/setup-env
...
+### Test
+
+```sh
+# Run all the packages tests
+$ yarn test
+
+# Run the "code-du-travail-frontend" tests
+$ yarn workspace code-du-travail-frontend test
+
+# Build the "<package-name>"
```
Comment (lionelB) [inline README.md]: this is probably a copy-paste typo

**Review by lionelB — CHANGES_REQUESTED** [review]
Merci @douglasduteil pour le howto

File: README.md (hunk 1 above)
Comment (douglasduteil) [reply README.md:50]: done

**Comment by douglasduteil** [comment]
:up:

**Review by revolunet — APPROVED** [review]

**Pull request merged**